tests/integration_tests/tests/agentless_tests/test_events.py

```diff
@@ -135,6 +140,31 @@ def test_events_with_logs(self):
         else:
             self.fail("Expected logs to be found")
 
+    def test_snapshots_events(self):
+        """ Make sure snapshots events appear when using the
+         cfy events list command """
+        # Make sure 'snapshots create' events appear
+        self.client.snapshots.create('id_0', False, False)
+        time.sleep(20)
```

Comment:
Use wait for `wait_for_execution_to_end` instead (https://github.com/cloudify-cosmo/cloudify-manager/blob/master/tests/integration_tests/tests/test_cases.py#L395)